pkg/controller/networkpolicy/validate_test.go

```diff
@@ -653,6 +653,105 @@ func TestValidateAntreaClusterNetworkPolicy(t *testing.T) {
 			operation:      admv1.Create,
 			expectedReason: "group cannot be set with other peers in rules",
 		},
+		{
+			name: "acnp-rule-ipblock-invalid-ipv4-cidr",
```

Comment:
here as well, we can keep these 2 tests but this should be guaranteed by the `cidr` format in the OpenAPI spec?